include/libcyphal/types.hpp

```diff
@@ -57,6 +57,12 @@ template <typename Interface, typename Any>
 class ImplementationCell final
 {
 public:
+    ImplementationCell()
+        : fn_getter_mut_{[](Any&) -> Interface* { return nullptr; }}
+        , fn_getter_const_{[](const Any&) -> const Interface* { return nullptr; }}
+    {
+    }
+
```

Comment:
Is there no way to avoid this? The absence of the default ctor is advantageous because it reduces the risk of having an instance of the implementation cell in an invalid state. The only way to get it to be invalid is to move the implementation out of it, which is only possible if Any is movable.

Reply:
I'll take a look.

Reply:
deleted - thank you for noticing!